src/components/GoalItem.tsx

```diff
@@ -0,0 +1,142 @@
+import styled, { css } from 'styled-components';
+import { useIntl, useTranslations } from 'next-intl';
+
+import type { Scalars, State, Tag, UserAnyKind } from '../../graphql/@generated/genql';
+import { goalBackgroundHover, textColor } from '../design/@generated/themes';
+
+import { Tag as TagItem } from './Tag';
+import { Icon } from './Icon';
+import { UserPic } from './UserPic';
+import { StateDot } from './StateDot';
+
+interface GoalItemProps {
+    title: string;
+    issuer?: UserAnyKind;
+    id: string;
+    tags?: Tag[];
+    state?: State;
+    createdAt: Scalars['DateTime'];
+    owner?: UserAnyKind;
+    comments?: number;
+    hasForks?: boolean;
+    isNotViewed?: boolean;
+}
+
+const StyledGoal = styled.a`
+    display: grid;
+    grid-template-columns: 15px 30px 600px repeat(3, 70px);
+    align-items: center;
+    color: ${textColor};
+    text-decoration: none;
+    &:hover {
+        background-color: ${goalBackgroundHover};
+    }
+    &:visited {
+        color: ${textColor};
+    }
+
+    padding: 10px 24px;
+`;
+
+const StyledState = styled.div`
+    align-self: center;
+    justify-self: center;
+`;
+
+const StyledNotViewed = styled.div`
+    align-self: center;
+    justify-self: center;
+`;
+
+const StyledNotViewedDot = styled.div`
+    width: 5px;
+    height: 5px;
+
+    background-color: ${textColor};
+
+    border-radius: 100%;
+`;
+
+const StyledName = styled.div`
+    width: 800px;
+    max-width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+`;
+
+const StyledTitle = styled.div`
```

Comment:
We have declared rest of text styles. Would be better to use on of them https://github.com/taskany-inc/issues/blob/main/src/components/Text.tsx

Reply:
done